backend/src/services/mail_verification/send.service.ts

```diff
@@ -0,0 +1,42 @@
+import { ENV } from '@common/enums/app/app';
+import nodemailer from 'nodemailer';
+
+import { generateTokens } from './token.service';
+
+const transporter = nodemailer.createTransport({
+  host: ENV.MAIL.SMTP_HOST,
+  port: ENV.MAIL.SMTP_PORT,
+  secure: false,
+  auth: {
+    user: ENV.MAIL.SMTP_USER,
+    pass: ENV.MAIL.SMTP_PASSWORD,
+  },
+});
+
+async function sendActivationMail(to: string, link: string): Promise<void> {
+  await transporter.sendMail({
+    from: ENV.MAIL.SMTP_USER,
```

Comment:
I as Administrator want to change mail from autoline.binary@gmail.com to no-reply@autoline.com for specific cases.
Long story short: 
add "from" to function params